Lib/json/decoder.py

```diff
@@ -161,7 +161,7 @@ def JSONObject(s_and_end, strict, scan_once, object_hook, object_pairs_hook,
             return pairs, end + 1
         elif nextchar != '"':
             raise JSONDecodeError(
-                "Expecting property name enclosed in double quotes", s, end)
+                "Expecting a property name enclosed in double quotes (no trailing comma allowed)", s, end)
```

Comment:
Is there a reason to not have this be handled in a dedicated case with a more specific error message? It kind of seems like we might be burying the lede with the current error message if the trailing comma is the issue.